File: pdf-editor-app-frontend/src/app/models/signature.ts
export class Signature {
  private _name: string;
  private _surname: string;

  constructor(name: string, surname: string) {
    this._name = name;
    this._surname = surname;
  }

  public toString():string {
    return `${this._name} ${this._surname}`;
  }
}

File: pdf-editor-rest/src/main/java/com/pdfeditor/rest/Signature.java
package com.pdfeditor.rest;

import lombok.Data;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;

@Data
@Entity
class Signature {

    private @Id @GeneratedValue Long id;
    private String name;
    private String surname;

    Signature() {}

    Signature(String name, String surname) {
        this.name = name;
        this.surname = surname;
    }
}